skillful_network_client/src/app/login/login.component.ts

```diff
@@ -132,4 +123,15 @@ export class LoginComponent implements OnInit {
     // can't event.preventDefault();
     console.log('onChange event.checked ' + event.checked);
   }
+
+  snackBarNewCodeSend() {
+    this._snackBar.open("Vous avez reçu un nouveau code", "X", {
+      duration: 1500,
+      verticalPosition: 'bottom'
+    });
+  }
+
+  reloadPage() {
+    setTimeout(function () { window.location.reload(); }, 3000);
```

Comment:
Is it necessary to reload the page? Could we reinit it?

Reply:
Maybe like this : I replaced method onload by ngOnInit() to reinit and I removed the condition of [disabled]= "submitted" from the button on this form.

commit [replace reload to ngOnInit()](https://github.com/danglotb/skillful_network/pull/45/commits/b4909e4ac407def65bdd1ca079baa7c83c087f90)